### DISCUSSION TITLE
Cachebench config to specify time

### DISCUSSION THREAD

**Discussion by wangyuyue** [facebook/CacheLib#214] [General]
Hi developers,
I want to test the cachelib performance for SSDPerf benchmarks provided in the repo. Instead of specifying a number of operations per thread, I want to specify a time duration. Is there any built-in configuration option for that?

Thanks,
Yuyue

**Comment by jaesoo-fb** [comment]
Hi @wangyuyue,

For the purpose you mentioned, cachebench provides the command line option `-timeout_seconds`; see `cachebench --help` for details. Please note that the test would stop once whichever shorter one among `numOps` and `timeout` is reached.

Thanks.

**Comment by wangyuyue** [comment]
Thanks, problem solved.
